### PR TITLE
#7 Added support for custom input

### PR DESCRIPTION
**Pull request by adarsh1021**
For #7 
Added a new textarea for passing custom inputs. White spaces and new lines are removed and the string is converted into an array.
The custom inputs are displayed in log as seen below.

I have also added this text area to the clear function. So on clicking the clear button both the code and custom input are cleared.

### CODE REVIEW

**Comment by championswimmer** [comment]
@omerjerk please verify

File: scripts/main.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ $(document).ready(function () {
         }
 
         source = window.btoa(source);
-        var testcases = ''; //hardcoded for now
+        var testcases = $("#test-input").val().replace( /\n/g, " " ).split( " " ); // cusotm inputs
+		console.log(testcases); // logs custom inputs
         var expected = '';
         for (var i = 0; i < expected.length; ++i) {
```
Comment (omerjerk) [inline scripts/main.js]: @adarsh1021 
Remove this for loop. And assign the textarea value to the `testcases` variable.

No need to convert the text to array. Just assign the whole value to that testcases variable.

File: scripts/main.js [hunk 2]
```diff
@@ -85,6 +86,7 @@ $(document).ready(function () {
 
     $('#clear').click(function () {
         ace.edit("editor").setValue('');
+		document.getElementById('test-input').value = "";
```
Comment (omerjerk) [inline scripts/main.js]: Please fix the indentation here.

File: scripts/main.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ $(document).ready(function () {
     runButton.click(function () {
         runButton.button('loading');
         var source = ace.edit("editor").getValue();
-
+
```
Comment (omerjerk) [inline scripts/main.js]: Why did you add so many spaces?
It's really important in open source to keep whitespaces in limit. ;)
If you want to add a new line, that's fine, but don't add so many whitespaces.

Comment (adarsh1021) [inline scripts/main.js]: Thank you for the feedback! I will keep that in mind :)

File: scripts/main.js [hunk 4]
```diff
@@ -49,12 +48,10 @@ $(document).ready(function () {
         }
 
         source = window.btoa(source);
-        var testcases = ''; //hardcoded for now
+        var testcases = $("#test-input").val(); // cusotm inputs
+		console.log(testcases); // logs custom inputs
```
Comment (omerjerk) [inline scripts/main.js]: @adarsh1021 
Remove this `console.log` line. Also add base64 encode the `testcases` variable. For e.g.
`testcases = window.btoa(testcases);`

**Comment by abhishek97** [comment]
@adarsh1021 Please take care of custom input for `js` as well.(it's implemented as `eval` inside browser. One idea for this can be to provide a global `_input` as string containing input and maybe some helper functions (like `get_line()` or `getNum()` ) etc

**Comment by omerjerk** [comment]
@adarsh1021 fix the merge conflicts for now and your code is good to be merged. Thanks a lot.
